app/javascript/packages/document-capture/components/review-issues-step.tsx

```diff
@@ -98,8 +110,35 @@ function ReviewIssuesStep({
     if (pii) {
       return <BarcodeAttentionWarning onDismiss={onWarningPageDismissed} pii={pii} />;
     }
-
+    // ipp disabled
     if (!inPersonURL || isFailedResult) {
+      if (isFailedDocType) {
```

Comment:
What is different about the rendered value of these branching paths than the default? I notice there's a lot of repetition in this file, and wonder if we could make those differences more targeted, so that the file is easier to read and maintain. (Personally, I've already had issues with this file previously since the repetition has caused me to look at the wrong code)

Reply:
@aduth yes, there are quite some repetition of the code, we have a following card to audit the code and clean it up.

The main difference is the error message generated.

Reply:
@dawei-nava Which ticket is it? I'd be interested to keep an eye on it.

Reply:
@aduth, refactor it anyway, mentioned ticket is LG-10363.